### PR TITLE
Added DevOps to roles-in-tech

### PR DESCRIPTION
**Pull request by pydevsg**
A short description about Devops field.
Fixed Issue #93.

### CODE REVIEW

File: roles-in-tech/dev-ops.md [hunk 1]
```diff
@@ -17,75 +13,82 @@ Here are links to lessons that should be completed before this lesson.
...
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
+[AWS Certified DevOps Engineer - Professional](https://aws.amazon.com/certification/certified-devops-engineer-professional/)
 
 
 ### Check for Understanding
 
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
+For Better Understanding of *DevOps* , one should do continuos unit testing after deploying it on a suitable platform.
```
Comment (manaswinidas) [inline roles-in-tech/dev-ops.md]: continuous

**Comment by pydevsg** [comment]
Any other changes you would like to see @manaswinidas.
Thanks for the review😊.

**Comment by alodahl** [comment]
Also, I like that you are including images!  Instead of putting images in the outline, could you please put them in a slideshow version of your outline here?  It should help visual learners with the topic. https://docs.google.com/presentation/d/1GaoHCdvQ2UplDjapp2wG-VRsxlqtzFqIpOtIqvKNYOo/edit?usp=sharing  Please consult these slide guidelines for styling: #600

**Comment by pydevsg** [comment]
@alodahl . you mean instead of adding photos I should add a ppt file within my DevOps file? Is it or I should add photos as well as ensure that it is being presented in the ppt file too?

**Comment by alodahl** [comment]
@pydevsgplease have no images in the md and all the images in this slides link. Yes, please add the slides to the “materials” section.

**Comment by pydevsg** [comment]
I need some help.
Why am I getting this error while pushing my commits.
>git push origin newbranch
remote: Permission to Techtonica/curriculum.git denied to pydevsg.
fatal: unable to access 'https://github.com/Techtonica/curriculum.git/': The requested URL returned error: 403

**Comment by pydevsg** [comment]
Please review [661](https://github.com/Techtonica/curriculum/pull/661) @alodahl and @manaswinidas

**Comment by alodahl** [comment]
@pydevsg, if you’re happy with your new PR, you can close this one.

**Comment by pydevsg** [comment]
I am closing this PR, @alodahl .

**Pull request closed without merging**